src/rocker/build/client.go

```diff
@@ -310,6 +310,10 @@ func (c *DockerClient) RunContainer(containerID string, attachStdin bool) error
 
 	signal.Notify(sigch, os.Interrupt)
 
+	/* Signal handler should be reset right after exit of this scope.
+	We don't want this sighanler to stay alive and suppress default signal handler if any*/
```

Comment:
Minor thing. Can you please change this comment to use `//` on each line. It is not documented in [Effective go](https://golang.org/doc/effective_go.html#commentary) guide, but all of the Golang code I've seen do not use multiline comment blocks in places except package and function headers.